rose/server/game.py

```diff
@@ -43,8 +42,8 @@ def start(self):
         if self.started:
             raise error.GameAlreadyStarted()
         self.track.reset()
-        for player in self.players.values():
-            player.reset()
+        for actor in self.players.values():
+            actor.reset()
```

Comment:
We don't use the term "actor" in the project. To avoid the shadowing, we can just use "p" instead:

    for p in self.players.values():
        p.reset()